pkg/cmd/setup_apps.go

```diff
@@ -97,7 +97,8 @@ var appsCmd = &cobra.Command{
 		}
 
 		err = execActionIfEnabled(actions.Actions.ConfigureCstorPool, func() error {
-			err = k8s.CreateCStorPoolClusters(captenConfig)
+
+			err = k8s.CreateCStorPoolClusterWithRetries(captenConfig)
 			if err != nil {
 				return errors.WithMessage(err, "failed to configure storage pool")
```

Comment:
don't return error,  log it

clog.Logger.Errorf("Failed to configure storage pool, %v", err)
return